### ISSUE TITLE
Support for Symfony 5.4 is broken

### ISSUE THREAD

**Issue by chrif** [osteel/openapi-httpfoundation-testing#35]
**Package version**
0.10

**Describe the bug**
I cannot update to 0.10 because of a new requirement on symfony/cache ^6.0

**To Reproduce**
composer.json 
```
{
  "require": {
    "php": ">=8.2",
    "symfony/symfony": "5.4.*",
  },
  "require-dev": {
    "osteel/openapi-httpfoundation-testing": "^0.9.0",
  },
  "extra": {
    "symfony": {
        "require": "5.4.*"
    },
}
```
```
$ composer require --dev osteel/openapi-httpfoundation-testing:^0.10.0 -W
./composer.json has been updated
Running composer update osteel/openapi-httpfoundation-testing --with-all-dependencies
Loading composer repositories with package information                                                                                                                       Info from https://repo.packagist.org: #StandWithUkraine
Restricting packages listed in "symfony/symfony" to "5.4.*"
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires osteel/openapi-httpfoundation-testing ^0.10 -> satisfiable by osteel/openapi-httpfoundation-testing[v0.10].
    - osteel/openapi-httpfoundation-testing v0.10 requires symfony/cache ^6.0 -> found symfony/cache[v6.0.0, ..., v6.3.0] but these were not loaded, likely because it conflicts with another require.
```

**Expected behaviour**
I expected the library to be compatible with maintained Symfony branches, which are 5.4, 6.2, and 6.3: https://symfony.com/releases

**Comment by osteel** [comment]
Hey there!

Do you mind installing branch `feature/dependency-support` and let me know if that fixes it?

```shell
$ composer require --dev osteel/openapi-httpfoundation-testing:dev-feature/dependency-support
```

**Comment by chrif** [comment]
Yes it's working with the branch. Thank you.

**Comment by osteel** [comment]
Fix available as of [v0.10.1](https://github.com/osteel/openapi-httpfoundation-testing/releases/tag/v0.10.1). Thanks for reporting!
